src/bindings/jna/libelektra/src/main/java/org/libelektra/Elektra.java

```diff
@@ -237,7 +237,8 @@ int elektraGOptsContractFromStrings (Pointer contractKeySet, long argsSize, Stri
 	 *
 	 * @see #keyDel(Pointer) keyDel()
 	 *
-	 * @param name A valid name to the key, or {@link #KEY_NAME_SIMPLE_INITALIZED_BUT_EMPTY_OBJECT} to get a
+	 * @param name A valid name to the key, or
+	 *             {@link #KEY_NAME_SIMPLE_INITALIZED_BUT_EMPTY_OBJECT} to get a
```

Comment:
This sentence is confusing, it somehow implies that KEY_NAME_SIMPLE_INITALIZED_BUT_EMPTY_OBJECT is not valid. Do we still need this long alias for the cascading root key?

Reply:
Fixed documentation and renamed to CASCADING_ROOT_KEY_NAME